MeshLib/MeshSubset.h

```diff
@@ -91,6 +103,25 @@ class MeshSubset
         return _msh.getElements().cend();
     }
 
+    MeshSubset*
+    setIntersectionByNodes(std::vector<Node*> const& nodes) const
```

Comment:
@TomFischer No, the assumption can not be made. One could sort the input `nodes` internally, though. But for currently planned usage it will not change anything significant.
To use `std::set_intersection` both ranges must be sorted, which is pulling some more changes.